app/loader.cpp

```diff
@@ -120,9 +120,17 @@ bool Loader::forceLoad( const QString &filePath, bool force )
     emit projectReloaded( mProject );
   }
 
+  QMap<QString, QgsMapLayer *> layersMap = mProject->mapLayers();
+  QStringList invalidLayers;
+  for ( QgsMapLayer *layer : layersMap.values() )
+  {
+    if ( !layer->isValid() )
+      invalidLayers.push_back( layer->name() );
```

Comment:
maybe it would be worth to add it to the logger for debugging @wonder-sk ? `CoreUtils::warn( "Input", msg );`